tests/microbenchmarks.py

```diff
@@ -169,6 +176,7 @@ def test_cmp(test_case):
                 x = [3.0] > [5.0]
     """), timed=True)
 
+
 def test_loops(test_case):
     print("Running", "test_loops")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils